LXReorderableCollectionViewFlowLayout/LXReorderableCollectionViewFlowLayout.m

```diff
@@ -163,7 +168,7 @@ - (void)invalidateLayoutIfNecessary {
     if ([self.dataSource respondsToSelector:@selector(collectionView:itemAtIndexPath:willMoveToIndexPath:)]) {
         [self.dataSource collectionView:self.collectionView itemAtIndexPath:previousIndexPath willMoveToIndexPath:newIndexPath];
     }
-
+    
```

Comment:
These should be removed ideally.   There's an Xcode preferences setting that will cause these not to happen.